inc/commondbtm.class.php

```diff
@@ -36,64 +36,191 @@
    die("Sorry. You can't access this file directly");
 }
 
-
-
-
 /**
 *  Common DataBase Table Manager Class - Persistent Object
 **/
 class CommonDBTM extends CommonGLPI {
 
-   /// Data of the Item
-   public $fields                              = [];
-   /// Make an history of the changes
-   public $dohistory                           = false;
-   /// Black list fields for history log or date mod update
-   public $history_blacklist                   = [];
-   /// Set false to desactivate automatic message on action
-   public $auto_message_on_action              = true;
+   /**
+    * Data fields of the Item.
+    *
+    * @var mixed[]
+    */
+   public $fields = [];
+
+   /**
+    * Flag to determine whether or not changes must be logged into history.
+    *
+    * @var boolean
+    *
+    * @TODO This variable should not be updated and could be converted as a constant (protected if min PHP >= 7.1).
```

Comment:
As I've said, I do not agree to change that to constants. Those todo can be removed right now

Reply:
OK.

Reply:
I changed for another proposal. Can be refused too :).